tox.ini

```diff
@@ -82,7 +82,6 @@ extras =
 commands =
     !online-!hypothesis-!figure: {env:PYTEST_COMMAND} {posargs}
     online: {env:PYTEST_COMMAND} --reruns 2 --timeout=180 --remote-data=any -m "remote_data" {posargs}
-    devdeps-!figure: bash -ec "for f in {toxinidir}/examples/*/*.py; do [[ $f == *skip* ]] && continue; echo "$f" && python "$f"; done"
```

Comment:
Time to restore the devdeps as working now.

Reply:
Cool 👍